ts/Core/Axis/TreeGrid/TreeGridAxis.ts

```diff
@@ -328,12 +328,16 @@ function getTreeGridFromData(
             start: number,
             result: Record<string, GridNode>
         ): Record<string, GridNode> {
-            const nodes = gridNode.nodes,
+
+            const
+                nodes = gridNode.nodes,
```

Comment:
Just indentation. I believe we are not making enters (new lines) after `const` or `let`.

Reply:
Alright, but I am afraid to tell you, Hubert 😬 , I think I may have been doing this for a while 😆